care/emr/resources/medication/dispense/spec.py

```
@@ -170,8 +171,20 @@ def perform_extra_deserialization(self, is_update, obj):
             if not dispense_order_obj:
+                if (
+                    self.create_dispense_order.status is not None
+                    and self.create_dispense_order.status
+                    in [
+                        MedicationDispenseOrderStatusOptions.draft.value,
+                        MedicationDispenseOrderStatusOptions.in_progress.value,
+                    ]
+                ):
+                    raise ValidationError(
+                        "Prescription status must be draft or in_progress"
+                    )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)